fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -415,6 +428,7 @@ private Resource getSkolemizedResource(final IdentifierConverter<Resource, Fedor
     }
 
     private static String skolemizedId() {
-        return "/.well-known/genid/" + randomUUID();
+        return "/.well-known/genid/";
```

Comment:
This method isn't producing an actual identifier, but a prefix. Maybe the name can change to reflect that?